web/src/lib/utils/analytics.ts

```diff
@@ -2362,5 +2363,33 @@ export default {
         },
       },
     },
+    check_account_help_button: {
+      click: {
+        open_live_chat: () => {
+          eventRunner.track({
+            event: "link_clicks",
+            legacy_event_action: "click",
+            legacy_event_category: "check_account_help_button",
+            legacy_event_label: "open_live_chat",
+            click_text: "check_account_help_button",
+            clicked_to: "open_live_chat",
+          });
+        },
+      },
+    },
+    check_account_next_button: {
+      submit: {
+        go_to_myNJ_login: () => {
+          eventRunner.track({
+            event: "account_clicks",
+            legacy_event_action: "click",
+            legacy_event_category: "check_account_log_in",
+            legacy_event_label: "go_to_myNJ_login",
+            clicked: "go_to_myNJ_login",
+            item: "check_account_log_in",
+          });
+        },
+      },
+    },
```

Comment:
still waiting on final values for these from @alekvuka

Reply:
[got the go-ahead on these](https://www.pivotaltracker.com/story/show/188540268/comments/243310127), just need validate in dev/test now